services/navigation/builtin/builtin.go

```diff
@@ -55,10 +67,17 @@ type Config struct {
 	BaseName           string                 `json:"base"`
 	MovementSensorName string                 `json:"movement_sensor"`
 	MotionServiceName  string                 `json:"motion_service"`
+	VisionServices     []string               `json:"vision_services"`
+
 	// DegPerSec and MetersPerSec are targets and not hard limits on speed
-	DegPerSec    float64                          `json:"degs_per_sec"`
-	MetersPerSec float64                          `json:"meters_per_sec"`
-	Obstacles    []*spatialmath.GeoObstacleConfig `json:"obstacles,omitempty"`
+	DegPerSec    float64 `json:"degs_per_sec"`
+	MetersPerSec float64 `json:"meters_per_sec"`
```

Comment:
I think these two can become omit empty too

Reply:
sounds good.
In this case, do you think we should also make the `motion_service` omitempty as well? In the validate function we default the `motion_service` field to `"builtin"` if it is empty.